### PR TITLE
TRITON-2239 imgapi records wrong fingerprint for LXC images (filename independence)

### PR DESCRIPTION
**Pull request by bahamat**

### CODE REVIEW

**Review by danmcd — APPROVED** [review]
Hopefully this is what @twhiteman had in mind.  Seems that way...

File: lib/lxdclient.js [hunk 1]
```diff
@@ -314,14 +314,19 @@ function getImageBestFile(img, version) {
         // items but that is not guaranteed to be the case.
         // Discussing with the LXC folks, multiple files of the same ftype would
         // likely result in indeterministic behavior.
+        var itemFromFtype = {};
+
+        Object.keys(items).forEach(function(name) {
```
Comment (danmcd) [inline lib/lxdclient.js]: Build is complaining around here...

**Review by danmcd — CHANGES_REQUESTED** [review]

File: lib/lxdclient.js (hunk 1 above)
Comment (bahamat) [reply lib/lxdclient.js]: Yeah, just fixed that. New one on the way.

**Review by danmcd — APPROVED** [review]
Put testing notes in the ticket!

**Pull request merged**